scripts/us_census/pep/pep_by_asrh/preprocess.py

```diff
@@ -0,0 +1,1072 @@
+    }
+    # Removing SV"s from 5 to 10 as they are not part of origin
+    # HispanicOrlatino (or) NotHispanicOrLatino
+    data_df = data_df[~data_df["SV"].isin(["5", "6", "7", "8", "9", "10"]
+                                         )].reset_index(drop=True)
+    skipped_data_df = skipped_data_df[~skipped_data_df["SV"].isin(
+        ["5", "6", "7", "8", "9", "10"])].reset_index(drop=True)
+    data_df["SV"] = data_df["SV"].map(sv_mapper)
+    skipped_data_df["SV"] = skipped_data_df["SV"].map(sv_mapper)
+    derived_cols = _get_mapper_cols_dict("county_1900_1999")
+    data_df[pop_cols] = data_df[pop_cols].apply(pd.to_numeric, errors="coerce")
+    data = None
+    for dsv, sv in derived_cols.items():
+        data = data_df[data_df["SV"].isin(sv)].reset_index(drop=True)
+        data["SV"] = dsv
+        data = data.groupby(["Year", "Location", "SV"]).sum().reset_index()
+        data_df = pd.concat([data_df, data])
+    data_df = pd.concat([data_df, skipped_data_df])
+    data_df = data_df.dropna()
+
+    data_df = pd.melt(data_df,
+                      id_vars=["Year", "Location", "SV"],
+                      value_vars=pop_cols,
+                      var_name="Age",
+                      value_name="Count_Person")
+    # Creating SV"s name using SV, Age Column
+    data_df["SV"] = data_df.apply(lambda row: _create_sv(row.SV, row.Age),
+                                  axis=1)
+    # Creating GeoId"s for State FIPS Code
+    data_df["Location"] = "geoId/" + data_df["Location"].astype("str").str.pad(
+        width=5, side="left", fillchar="0")
+    # Deriving Measurement Method for the SV"s
+    data_df = _add_measurement_method(data_df, "SV", "Measurement_Method")
+    data_df = data_df[[
+        "Year", "Location", "SV", "Measurement_Method", "Count_Person"
+    ]]
+    return data_df
+
+
+def _process_county_2000_2009(file_path: str) -> pd.DataFrame:
+    """
+    Returns the Cleaned DataFrame consists
+    county data for the year 2000-2009.
+
+    Args:
+        file_path (str): Input File Path
+
+    Returns:
+        pd.DataFrame: Cleaned DataFrame
+    """
+    data_df = _load_data_df(path=file_path,
+                            file_format="csv",
+                            header=0,
+                            encoding="latin-1")
+    # Skipping Below Year Values
+    # 1 = 4/1/2000 resident population estimates base
+    # 12 = 4/1/2010 resident 2010 Census population
+    # 13 = 7/1/2010 resident population estimate
+    data_df = data_df[(~data_df["YEAR"].isin([1, 12, 13])) &
+                      (data_df["AGEGRP"] != 99)].reset_index(drop=True)
+    data_df["YEAR"] = 1998 + data_df["YEAR"]
+    cols = list(data_df.columns)
+    # Mapping Dataset Headers to its FullForm
+    cols_mapper = _get_mapper_cols_dict("header_mappers")
+    for idx, val in enumerate(cols):
+        cols[idx] = cols_mapper.get(val, val)
+    data_df.columns = cols
+    data_df["STATE"] = data_df["STATE"].astype("str").str.pad(width=2,
```

Comment:
The pad() function is being used several times in this file. Could you add comments to explain the use wherever the comments do not exist?